gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -54,9 +54,7 @@ class VeloxTestSettings extends BackendTestSettings {
       "SPARK-32038: NormalizeFloatingNumbers should work on distinct aggregate",
       // Replaced with another test.
       "SPARK-19471: AggregationIterator does not initialize the generated result projection" +
-        " before using it",
```

Comment:
Could you also test whether the test case for "SPARK-31993: concat_ws in agg function with plenty of string/array types columns" can now be included?

Reply:
thank you for the reminder, enable that test